packages/ui/src/MultiselectField.tsx

```diff
@@ -3,16 +3,20 @@ import {TouchableOpacity, View} from "react-native";
 
 import {CheckBox} from "./CheckBox";
 import {MultiselectFieldProps} from "./Common";
+import {FieldError, FieldHelperText} from "./FieldElements";
 import {Heading} from "./Heading";
 import {isMobileDevice} from "./MediaQuery";
 import {Text} from "./Text";
 
-const Option: FC<{
+interface OptionProps {
   isDefault: boolean;
-  option: string;
+  value: string;
+  title?: string;
```

Comment:
Would it make more sense to call this label, since it comes from the label property for the option? We use title as more of a heading for the rest of the components. Feels to me like we should just keep it consistent since we're just passing it down here from the option where we call it label in the first place.

Reply:
Yup makes sense!